tool/state-processor/src/type_registry.rs

```diff
@@ -4,26 +4,35 @@ use std::iter::once;
 use enumflags2::{bitflags, BitFlags};
 use parity_scale_codec::{Decode, Encode, EncodeLike, Error, Input};
 
+pub type Balance = u128;
+pub type AssetBalance = u128;
```

Comment:
I think `Balance` is enough.

Reply:
The underlying types are the same. But they are represent two different things, one for the native currency, one for the asset currentcy. So, I think we can split them into two types to make it more clearer. Do you still think it is unnecessary?

Reply:
The underlying types are the same. But they are represent two different things, one for the native currency, one for the asset currentcy. So, I think maybe we can split them into two types to make it more clearer.